modules/microshift-install-rhel-image-mode-cni.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-run-image-cni_{context}"]
+= Checking the {microshift-short} bootc image for the CNI
+
+The {microshift-short} OVN-Kubernetes (OVN-K) Container Network Interface (CNI) requires the Open vSwitch service. The `openvswitch` kernel module must be available in the `bootc` image.
+//can we include this in the containerfile?
```

Comment:
No, because we do not know the kernel version which will be used to run the container.